# max rate of the planner
RATE: 25
# goal tolerance
XY_GOAL_TOLERANCE: 0.04
YAW_GOAL_TOLERANCE: 0.15
# maximum speed
V_MAX: 0.25 #0.35
W_MAX: 1.5
# acceleration at beginning of movements
ACCELERATION: 1.0
# goal distance for deceleration
D_DECELERATION: 0.15 #0.25
# power of deceleration factor
GAMMA: 0.75
# params for accurate movements to water towers or cube heaps
XY_ACCURATE_GOAL_TOLERANCE: 0.005
YAW_ACCURATE_GOAL_TOLERANCE: 0.05
D_ACCURATE_DECELERATION: 0.25